solution/app/src/main/java/com/google/codelabs/maps/placesdemo/CurrentPlaceActivity.kt

```diff
@@ -167,9 +183,33 @@ class CurrentPlaceActivity : AppCompatActivity() {
         }
     }
 
+    override fun onMapReady(map: GoogleMap) {
+        this.map = map
+    }
+
+    private fun setPlaceOnMap(place: Place) {
+        try {
+            val latLng = place.latLng ?: defaultLocation
+            map?.moveCamera(
+                CameraUpdateFactory.newLatLngZoom(
+                    latLng,
+                    DEFAULT_ZOOM.toFloat()
+                )
+            )
+            map?.addMarker(
+            MarkerOptions()
+                .position(latLng)
+                .title(place.name)
+            )
+        } catch(e: SecurityException) {
```

Comment:
What was causing this SecurityException?  Seems weird to bury the exception here.